rust/agama-logs/src/main.rs

```diff
@@ -0,0 +1,221 @@
+extern crate fs_extra;
+extern crate tempdir;
```

Comment:
Please, remove the `extern crate` lines. That's something from the past :-)

Reply:
I believe that I read somewhere that it still is good to use ... but i don't have the article at hand and it easily could be five years old one ;-) so removed.